storage/testdb/testdb.go

```diff
@@ -92,7 +113,7 @@ func newEmptyDB(ctx context.Context) (*sql.DB, func(context.Context), error) {
 	}
 
 	db.Close()
-	db, err = sql.Open("mysql", *dataSourceURI+name)
+	db, err = sql.Open("mysql", mysqlURI()+name)
```

Comment:
Consider adding "/" here, and removing it from the address flag.

Reply:
done